metricbeat/module/system/process/_meta/fields.yml

```
@@ -398,6 +547,13 @@
                 The number of times that the memory limit (kmem_tcp.limit.bytes)
                 was reached.
 
+            - name: stats.*
+              type: nested
```

Are you sure you want to use [nested](https://www.elastic.co/guide/en/elasticsearch/reference/current/nested.html) here? `object` is probably fine, same for the rest of nested fields defined here.
```suggestion
              type: object
              object_type: float
```

---

Honestly I spent what felt like half a day trying to make the `fields.yml` happy, and I have no idea if I should use `groups` or `nested` or what. I can change it.